jobs/build/appregistry/appregistry.groovy

```diff
@@ -282,13 +282,48 @@ def stagePushDevMetadata(operatorBuilds) {
     def token = retrieveBotToken()
     def metadata_nvrs = getMetadataNVRs(operatorBuilds.collect { it.nvr }, "dev")
     def errors = new ConcurrentHashMap()
-    parallel metadata_nvrs.collectEntries { nvr -> [
+    def distgits = operatorBuilds.collect { it.distgit }
+    def owners = buildlib.get_owners("--working-dir ${workDir} -g openshift-${buildVersion}", distgits)
+
+    parallel [metadata_nvrs, distgits].transpose().collectEntries { nvr, distgit -> [
```

Comment:
it took me a bit to realize what this was doing... it's clever, perhaps too clever... any way this can be clearer? just... too much punctuation with different meanings on these two lines.

Reply:
I did a little refactor to make things clear. Could you have another look?